addons/website_sale/static/src/js/website_sale.js

```diff
@@ -1,4 +1,22 @@
 $(document).ready(function () {
+
+if($('ul').is('#collapse_categ')){
+    $('#collapse_categ').find('ul').addClass('collapse');
+    $('#collapse_categ').find('li.active').parents('ul').addClass('in').removeClass('collapse');
+    $('#collapse_categ').find('.active > ul').addClass('in').removeClass('collapse');
+    $('#collapse_categ').find('.parent-categ').prepend('<span class="fa fa-chevron-right"></span>');
+    $('#collapse_categ').find('ul.in').parent().find('a span')
+        .removeClass('fa-chevron-right')
+        .addClass('fa-chevron-down');
+    $('#collapse_categ').find('ul.collapse').parent().find('a span')
+        .removeClass('fa-chevron-down')
+        .addClass('fa-chevron-right');
+    $('#collapse_categ').find('.active a span').addClass('fa-chevron-down');
+    $('#collapse_categ').find('.active').children('ul').find('span')
+        .removeClass('fa-chevron-down')
+        .addClass('fa-chevron-right');
+}
```

Comment:
Too mush add/remove class in js.
You should add the `fa` icon and all the class in the qweb template. The role of javascript will only be to animate the accordeon effect, by clicking on `fa-chevron`, and expand the parent category of the current one. (We want to add : a click on 'chevron' expand/collapse the category `ul` list. When clicking on the link, it redirect to the correct page).
You can be inspired by http://bootsnipp.com/snippets/featured/sidebar-menu-group-tree (keep your css style).